hutch_python/ophyd_settings.py

```diff
@@ -0,0 +1,29 @@
+"""
+Module to contain ophyd-specific settings that must be applied prior to
+creating any ophyd objects.
+"""
+from ophyd.signal import EpicsSignalBase
+
+
+def setup_ophyd():
+    """
+    Call EpicsSignalBase.set_defaults with pcds defaults.
+
+    Sets the following to the current ophyd defaults:
+      - connection_timeout
+      - timeout (read timeout)
+      - auto_monitor
+
+    Overriding defaults with themselves seems redundant, but I want to
+    be insulated from changes to the upstream consensus.
+
+    Sets a new default to the following:
+      - write timeout (we don't want to wait forever)
+    """
+    EpicsSignalBase.set_defaults(
+        connection_timeout=1.0,
```

Comment:
Just a thought:
Would you consider making each of these an environment variable, such that we could tweak per-environment settings if this ends up being an issue on a larger scale than expected?

I'd imagine this will only affect 1 or 2 devices in a given hutch, but say for the sake of the argument it ends up affects like 50 of them. Then it starts to make more sense to just increase the environment-specific settings (or wait until hutch-python gets updated).

`HUTCH_PYTHON_CONNECTION_TIMEOUT=1.0` and whatnot

Reply:
This is easy enough to do that it's worth doing

Reply:
I now recall that we've occasionally had issues (in various apps) with e.g. slow waveforms from the gateway, which can be solved by extending the read timeout. Therefore, being able to manipulate this via environment variables already has an observed use case.